extensions/git/src/timelineProvider.ts

```diff
@@ -58,8 +58,8 @@ export class GitTimelineItem extends TimelineItem {
 }
 
 export class GitTimelineProvider implements TimelineProvider {
-	private _onDidChange = new EventEmitter<TimelineChangeEvent>();
-	get onDidChange(): Event<TimelineChangeEvent> {
+	private _onDidChange = new EventEmitter<TimelineChangeEvent | undefined>();
```

Comment:
This change is fixing a compile error and is just taking over what VS Code has already done in their repo.